### DISCUSSION TITLE
Reading Ethereum + Solidity Official Documentation

### DISCUSSION THREAD

**Discussion by 0x1057** [Cyfrin/foundry-full-course-cu#3065] [General]
I was just curious if anyone has found benefit in reading these two sets of documentation. I don't mean referencing them when needed. I mean have you found benefit in taking a day or two and going through them in detail from start to finish? If so, how has it benefited you? And finally, would reading Solidity-by-Example be a fine substitution for the solidity docs at this point?  Thanks in advance!

**Comment by EngrPips** [comment]
Hello @0x1057, You know most of the studying that you do will get to count at a later date. Hardly is anything gratified instantly when it comes to knowledge seeking, as it is mostly delayed gratification, but it used to be a great reward whenever it came. To cut the long story short if you have the time and privilege to study it then please do and don't see it as a waste of time.

**Reply by 0x1057** [reply]
Thank you so much!

**Reply by EngrPips** [reply]
Anytime
